BP/localization/english/religion/bp_religion_paganism_l_english.yml

```diff
@@ -459,10 +489,11 @@ l_english:
  basque_house_of_worship_plural:0 "Temple"
 
  # Basque
- vasconic_pagan:0 "Basque"
- vasconic_pagan_adj:0 "Basque"
- vasconic_pagan_adherent:0 "Basque"
- vasconic_pagan_adherent_plural:0 "Basques"
+ #Named for Sky God; see WtWSMS
```

Comment:
? In WtWSMS it is called Vasconic

Reply:
Sorry, I meant Urtzi was the high god for vasconic_pagans in WtWSMS; I've been trying to rename specific _faiths_ in a way that avoids just naming them after the culture they originated to decouple religion and culture (at least somewhat).

Reply:
Is this change acceptable, or should I revert to Vasconic?

Reply:
Well, I understand what you want to do, but I don't know to what extent the users will identify Urtzi and what importance that deity had within the Vasconic pantheon. So it is difficult to tell, especially since the name and very existence of Urtzi is apparently subject to controversy.

Reply:
Reverted localization to "Vasconic" again.